content/en/blog/2025/upgrade_from_3.5_to_3.6_issue.md

```diff
@@ -0,0 +1,73 @@
+---
+title: How to Prevent a Key Issue in the etcd 3.5 to 3.6 Upgrade - A Summary and Solution
```

Comment:
Revised title suggestion:  Read Before Upgrading to Etcd v3.6.0
alternately:
Prevent a v3.6.0 Upgrade-Blocking Issue

... in order to get more people to read it.

Reply:
> Revised title suggestion: Read Before Upgrading to Etcd v3.6.0

This isn't a generic upgrade guide or checklist. It's specific for this issue.



> Prevent a v3.6.0 Upgrade-Blocking Issue

It also works, but I prefer to the suggestion provided by @spzala in https://github.com/etcd-io/website/pull/974#discussion_r2012842937